### PR TITLE
Adding enigma2 docs

### PR DESCRIPTION
**Pull request by fbradyirl**
**Description:**
Adding enigma2 docs

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15799

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/media_player.enigma2.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+[OpenWebif](https://github.com/E2OpenPlugins/e2openplugin-OpenWebif) is an open source web interface for Enigma2 based set-top boxes.
+
+To add a set-top box to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: enigma2
+    host: 192.168.1.12
+    name: Vu Duo2
```
Comment (frenck) [inline source/_components/media_player.enigma2.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by fbradyirl — COMMENTED** [review]
@frenck agree with your comments. I’ve removed the optional config from sample.

**Review by frenck — APPROVED** [review]

File: source/_components/media_player.enigma2.markdown [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+To add a set-top box to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: enigma2
+    host: 192.168.1.12
+```
+
+{% configuration %}
```
Comment (tjorim) [inline source/_components/media_player.enigma2.markdown:30]: According to the code the network port can be configured as well.
If that is correct, add it to this section and mention the default (80).

**Review by tjorim — CHANGES_REQUESTED** [review]
@fbradyirl, only one question for you sir.


⚠️ The `ha_release` will need an update before merging this one in.

**Review by frenck — CHANGES_REQUESTED** [review]
You are right, @tjorim!
@fbradyirl, could you add the missing option? 👍

**Comment by cgarwood** [comment]
Parent PR went stale and was closed, closing this one.

**Pull request closed without merging**